packages/core/ratelimits/src/index.ts

```diff
@@ -38,32 +38,43 @@ const calculateLimits = (declaredTier: DeclaredTier) => {
   if (burst < 0) burst = 1000000 // currently using -1 to define unlimited
   if (quota < 0) quota = 1000000 // currently using -1 to define unlimited
 
-  return [burst, quota]
+  return { burst, quota }
 }
 
-const findTier = (provider: string, tier: number | string): DeclaredTier => {
-  provider = provider.toLowerCase()
-  const matchedTier = Number.isInteger(Number(tier))
-    ? Limits[provider]?.[Number(tier)]
-    : (Limits[provider] as DeclaredTier[])?.find((element) => element.tierName === tier)
-  if (matchedTier === undefined) {
-    console.log(
-      `[provider, tier]: [${provider}, ${tier}] doesn't match any provider spec in limits.json`,
-    )
+const findTier = (providerName: string, tier: number | string): DeclaredTier | undefined => {
+  const tierIndex = Number(tier)
+  const provider: DeclaredTier[] = Limits[providerName.toLowerCase()]
+  if (!provider) {
+    console.log(`Rate Limit: Provider with name ${providerName} not found`)
```

Comment:
Use out logger instead of `console.log`

Reply:
It will raise circular dependency problems...

Reply:
It shouldn't. Logger is internal to bootstrap now.

Reply:
He's right about this. Maybe we should move `ratelimit` stuff into `bootstrap`.

Reply:
There is probably no reason for any logging to happen in `ratelimits`, it's just a helper package. Whoever uses it should do the logging.